lib/tasks/disposable_domains.rake

```diff
@@ -2,6 +2,8 @@
 require 'csv'
 namespace :disposable_domains do
   task :load, %i[s3_url] => [:environment] do |_task, args|
+    ActiveRecord::Base.connection.execute 'SET statement_timeout = 200000'
+    # Need to increase statement timeout since command takes a long time. 
```

Comment:
Some small nitpicks:

- Since we use parentheses elsewhere in this file and in other tasks' assignment of the statement timeout, makes sense to use them here as well?
- I'd expect the comment to occur either before or on the same line as the thing it's commenting
- Looks like there was a stray trailing whitespace on the comment

```suggestion
    # Need to increase statement timeout since command takes a long time.
    ActiveRecord::Base.connection.execute('SET statement_timeout = 200000')
```

Reply:
since it loads the entire environment for this task, we can use our existing code here:
```ruby
Reports::BaseReport.transaction_with_timeout do
  DisposableDomain.insert_all(names.map { |name| { name: } })
end
```

Reply:
wait nevermind that code uses the `read_replica` .... scratch that